dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/domain/Server.java

```diff
@@ -60,10 +63,19 @@ public static Builder builder() {
    public abstract int memoryGb();
 
    @Nullable
-   public abstract List<Disk> disks();
+   public abstract NetworkInfo networkInfo();
+
+   @Nullable
```

Comment:
Remove nullable annotations from all lists that have presence enforced in the builder. Apply this pattern in general.